ct-app/postman/postman_tasks.py

```diff
@@ -31,6 +31,23 @@ class TaskStatus(Enum):
 app.autodiscover_tasks(force=True)
 
 
+async def channel_balance(api: HoprdAPIHelper, address: str) -> float:
+    """
+    Get the channel balance of a given address.
+    :param api: API helper instance.
+    :param address: Address to get the balance from.
+    :return: Channel balance of the address.
+    """
+    channels = await api.outgoing_channels(address)
+
+    channel = list(filter(lambda c: c.peerAddress == address, channels))
+
+    if len(channel) == 0:
+        return 0
+    else:
+        return int(channel[0].balance) / 1e18
```

Comment:
You could use `reduce` to make this a one liner: https://www.geeksforgeeks.org/reduce-in-python/

Reply:
Operation is quite simple, the perf improvement from using functools is super small. Could be done for the sake of code beauty actually. Will keep this in mind for my refactoring that is still ongoing